packages/webapp/src/containers/Notification/index.jsx

```diff
@@ -66,6 +66,7 @@ export default function NotificationPage() {
               />
             );
           })
+          .reverse()
```

Comment:
I don't think the Jira ticket gave a full description of the requirements. As a result, I don't think this will fix the problem. I've asked for confirmation in Jira.